facebook/Gemfile

```diff
@@ -49,6 +55,8 @@ group :development do
   # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
   gem 'web-console', '>= 3.3.0'
   gem 'rspec-rails', '~> 3.8'
+  gem 'factory_bot_rails'
+  gem 'capybara', '>= 2.2'
```

Comment:
You have this gem duplicated in line 68

Reply:
Thank you @efrapp for reviewing my PR. I will add the user index page and also fix the duplicates in the gemfile.